### DISCUSSION TITLE
is the attribute for "company" missing in lang/*/validation.php or am I the problem?

### DISCUSSION THREAD

**Discussion by mkocurek** [Laravel-Lang/common#203] [Q&A] [answered]
I was wondering why the only attribute (I'm currently using) wasn't translated for "company" - a quite common input field for forms as I would assume. After reviewing the 'attributes' array, I found all things but the key for 'company' - is this missing or did I miss something? 

However, after adding `'company' => 'Firmenname'` for the languages I'm using i.e. german into the **'attributes' array** part of the file: 'lang/de/validation.php' - the problem was solved.

**Comment by andrey-helldar** [comment]
The answer is very simple - the attribute list is not official for Laravel and anyone who wants to can easily add to it. Until now, there was no need to translate the word “company” ¯\_(ツ)_/¯

**Reply by andrey-helldar** [reply, marked as answer]
I've [added](https://github.com/Laravel-Lang/attributes/pull/993) a few more translation keys from the actual project, except for the word “company”. As soon as we accumulate enough changes, we will release a new version.

Thank you for your suggestion!
